src/xGadgetron/cGadgetron/gadgetron_data_containers.h

```diff
@@ -248,6 +260,10 @@ namespace sirf {
 			std::cerr << "AcquisitionsFile::set_acquisition not implemented yet, sorry\n";
 		}
 		virtual void append_acquisition(ISMRMRD::Acquisition& acq);
+
+		virtual void append_sptr_acquisition( gadgetron::shared_ptr<ISMRMRD::Acquisition> sptr_acqu){throw LocalisedException("Function undefined for sirf::AcuisitionsFile.", __FILE__, __LINE__);};
```

Comment:
rename to `append_acquisition_sptr`. or do we have other functions with similar names?
also split into 2 lines

Reply:
agree with Kris - all our methods returning shared pointers have sptr as suffix, and shared pointer properties as prefix